src/main.cpp

```diff
@@ -205,28 +195,28 @@ void app_main(void) {
     esp32time.setTime(now.Second(), now.Minute(), now.Hour(), now.Day(), now.Month(), now.Year());
     engineerDisplay.print("[v] " + rtc.getName() + " initialized, time in esp32 updated.\n");
   }
-  if (SD_ON) {
-    xTaskCreate(&write_sdcard_demo_task, "write_sdcard_demo_task", CONFIG_ESP_SYSTEM_EVENT_TASK_STACK_SIZE, NULL, 5, NULL);
-  }
+  // if (SD_ON) {
+  //   xTaskCreate(&write_sdcard_demo_task, "write_sdcard_demo_task", CONFIG_ESP_SYSTEM_EVENT_TASK_STACK_SIZE, NULL, 5, NULL);
+  // }
   if (INT_ON) {
```

Comment:
Why is this commented out? Shouldn't the sd card work now?

Reply:
removed, card works noch demo task necessary